week6/leeeunbin/src/components/ProductRow.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Products } from '../data-access';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faEdit, faTrash } from '@fortawesome/free-solid-svg-icons';
+
+interface Props {
+  product : Products;
+  onDelete : (product : Products) => void;
+  onEdit : (product : Products) => void;
+}
+
+const ProductRow : React.FC<Props> = ( {product, onDelete, onEdit} ) => {
+  return (
+    <tr>
+      <td style={{color: product.stocked ? 'color' : 'red'}}>{product.name}</td>
+      <td>{product.price}</td>
```

Comment:
```suggestion
      <td>{product.price}$</td>
```
원래 제공해드린 코드가 price 옆에 $가 같이 표시되는 구조였는데 은빈님이 이걸 바꾸셔서 그런것같아요.
테스트코드가 $이게 붙어있다는걸 전제로 하기때문에 통과가 안된 것 같습니다. 이것만 붙여주시면 테스트 잘 통과됩니다. 
미션 잘 수행해주셨습니다!